### PR TITLE
* Fixed lost call setDirty in EventDispatcher::updateListeners. (Crashed...

### PR DESCRIPTION
**Pull request by shrimpz**
Fixed lost call setDirty in EventDispatcher::updateListeners. (Crashed in Android)

### CODE REVIEW

**Comment by minggo** [comment]
@shrimpz 
Could you send a test case for it?
Thanks.

File: cocos/base/CCEventDispatcher.cpp [hunk 1]
```diff
@@ -1032,6 +1032,7 @@ void EventDispatcher::updateListeners(Event* event)
         
         if (sceneGraphPriorityListeners)
         {
+            bool isDirtyForsceneGraphPriorityListeners = false;
```
Comment (minggo) [inline cocos/base/CCEventDispatcher.cpp]: isDirtyForsceneGraphPriorityListeners -> isDirtyForSceneGraphPriorityListeners?

Comment (shrimpz) [inline cocos/base/CCEventDispatcher.cpp]: yes, thx, I want to fix it.

i will splite a test case from my project.

**Comment by minggo** [comment]
@shrimpz 
How to reproduce the issue?

**Comment by shrimpz** [comment]
void xxx( Scene\* scene, Label\* lbl )
{
    if( lbl )
    {
        lbl->removeFromParentAndCleanup( true );
        auto ed = Director::getInstance()->getEventDispatcher();
        ed->isEnabled();
    }

```
auto d = Director::getInstance();
unsigned char rawData[] = {
    0xEF, 0xBB, 0xBF, 0xE4, 0xBD, 0x9C, 0xE7, 0xBB, 0x88, 0xE4, 0xB8, 0xAD,
    0xE7, 0xBD, 0xAE, 0xE8, 0x87, 0xB3, 0xE6, 0x94, 0xAF, 0xE6, 0xAD, 0xA3,
    0xE5, 0x9C, 0xA8, 0xE8, 0xAF, 0xAD, 0xE4, 0xBA, 0x8E, 0xE7, 0x94, 0xB1,
    0xE7, 0x94, 0xA8, 0xE8, 0xAF, 0x91, 0xE8, 0xAE, 0xAE, 0xE5, 0xB7, 0xB2,
    0xE4, 0xBA, 0x9A, 0xE5, 0x8D, 0x8F, 0xE8, 0xAF, 0xA6, 0xE9, 0x97, 0xB2,
    0xE4, 0xB8, 0x8B, 0xE5, 0x8A, 0xA1, 0xE6, 0x97, 0xA0, 0xE6, 0x96, 0x87,
    0xE7, 0xBD, 0x91, 0xE8, 0xB7, 0xB3, 0xE4, 0xBD, 0x93, 0xE7, 0xB4, 0xA2,
    0xE9, 0x80, 0x9F, 0xE6, 0x90, 0x9C, 0xE9, 0x80, 0x81, 0xE6, 0x96, 0xAF,
    0xE6, 0x97, 0xB6, 0xE6, 0xB2, 0x99, 0xE5, 0xA1, 0x9E, 0xE8, 0xAF, 0xB7,
    0xE6, 0x83, 0x85, 0xE5, 0x99, 0xA8, 0xE9, 0x85, 0x8D, 0xE5, 0xB0, 0xBC,
    0xE8, 0x83, 0xBD, 0x0D, 0x0A, 0xE6, 0x85, 0xA2, 0xE7, 0xBB, 0x9C, 0xE7,
    0xBD, 0x97, 0xE8, 0xBF, 0x9E, 0xE5, 0x88, 0xA9, 0xE4, 0xBA, 0x86, 0xE5,
    0x85, 0xB0, 0xE6, 0x8B, 0x89, 0xE5, 0x86, 0xB5, 0xE7, 0xA9, 0xBA, 0xE5,
    0x85, 0x8B, 0xE5, 0x8F, 0xAF, 0xE5, 0xBC, 0x80, 0xE5, 0xA2, 0x83, 0xE8,
    0xBF, 0x9B, 0xE8, 0xA7, 0xA3, 0xE6, 0x8E, 0xA5, 0xE7, 0x96, 0x86, 0xE9,
    0x94, 0xAE, 0xE7, 0xAE, 0x80, 0xE5, 0x8A, 0xA0, 0xE5, 0x8D, 0xB3, 0xE6,
    0x9C, 0xBA, 0xE6, 0x88, 0x96, 0xE4, 0xBC, 0x9A, 0xE5, 0x9B, 0x9E, 0xE7,
    0x8E, 0xAF, 0xE8, 0xAF, 0x9D, 0xE8, 0xA1, 0x8C, 0xE5, 0x85, 0xB3, 0xE8,
    0xAF, 0xA5, 0xE6, 0x9C, 0x8D, 0xE5, 0x8F, 0x8D, 0xE7, 0xBF, 0xBB, 0xE5,
    0x8F, 0x91, 0xE5, 0xB0, 0x94, 0xE4, 0xBF, 0x84, 0xE6, 0x96, 0xAD, 0xE7,
    0xAB, 0xAF, 0xE7, 0x9A, 0x84, 0x0D, 0x0A, 0xE5, 0xBE, 0x97, 0xE6, 0xAD,
    0xA4, 0xE5, 0x90, 0xB9, 0xE8, 0x88, 0xB9, 0xE7, 0xA9, 0xBF, 0xE5, 0xA4,
    0x84, 0xE7, 0xA1, 0x80, 0xE9, 0x99, 0xA4, 0xE5, 0x88, 0x9D, 0xE5, 0x87,
    0xBA, 0xE8, 0x80, 0xBB, 0xE9, 0xBD, 0xBF, 0xE6, 0x8C, 0x81, 0xE8, 0xBF,
    0x9F, 0xE5, 0x90, 0x83, 0xE7, 0xA8, 0x8B, 0xE6, 0x88, 0x90, 0xE6, 0xB2,
    0x89, 0xE5, 0xBD, 0xBB, 0xE8, 0xBD, 0xA6, 0xE6, 0x9C, 0x9D, 0xE8, 0xB6,
    0x85, 0xE5, 0xB8, 0xB8, 0xE5, 0xB0, 0x9D, 0xE6, 0x9F, 0xA5, 0xE6, 0x9B,
    0xBE, 0xE6, 0x93, 0x8D, 0xE9, 0xA4, 0x90, 0xE5, 0x8F, 0x82, 0xE8, 0xB8,
    0xA9, 0xE6, 0x89, 0x8D, 0xE9, 0x83, 0xA8, 0xE6, 0xAD, 0xA5, 0xE4, 0xB8,
    0x8D, 0xE8, 0xA1, 0xA5, 0xE4, 0xBC, 0xAF, 0xE5, 0xB9, 0xB6, 0xE5, 0x85,
    0xB5, 0xE5, 0x88, 0xAB, 0xE8, 0xA1, 0xA8, 0x0D, 0x0A, 0xE6, 0xA0, 0x87,
    0xE9, 0x81, 0x8D, 0xE8, 0xBE, 0xB9, 0xE9, 0x97, 0xAD, 0xE5, 0xBF, 0x85,
    0xE7, 0xAC, 0x94, 0xE6, 0xAF, 0x94, 0xE9, 0x80, 0xBC, 0xE6, 0x9C, 0xAC,
    0xE8, 0xBE, 0x88, 0xE8, 0xA2, 0xAB, 0xE8, 0x83, 0x8C, 0xE5, 0xA4, 0x87,
    0xE5, 0x8C, 0x97, 0xE6, 0x8A, 0xA5, 0xE4, 0xBF, 0x9D, 0xE5, 0xB8, 0xAE,
    0xE7, 0x89, 0x88, 0xE6, 0x9D, 0xBF, 0xE8, 0x88, 0xAC, 0xE6, 0x8B, 0x9C,
    0xE6, 0x91, 0x86, 0xE7, 0x99, 0xBE, 0xE7, 0x99, 0xBD, 0xE5, 0x90, 0xA7,
    0xE6, 0x8A, 0x8A, 0xE7, 0x96, 0xA4, 0xE5, 0xB7, 0xB4, 0xE6, 0x8C, 0x89,
    0xE7, 0x88, 0xB1, 0xE9, 0x98, 0xBF, 0xE7, 0xA2, 0x8D, 0xE5, 0xAE, 0x89,
    0xE6, 0xA1, 0x88, 0xE6, 0x9A, 0x97, 0xE5, 0x85, 0xAB, 0xE7, 0xBD, 0xA2,
    0xE9, 0x9C, 0xB8, 0xE5, 0x8D, 0x8A, 0xE6, 0x89, 0xAE, 0x0D, 0x0A, 0xE6,
    0x8A, 0xB1, 0xE6, 0x82, 0xB2, 0xE5, 0x80, 0x8D, 0xE5, 0xA5, 0x94, 0xE6,
    0xAF, 0x95, 0xE7, 0xBC, 0x96, 0xE5, 0x8F, 0x98, 0xE4, 0xBE, 0xBF, 0xE6,
    0x92, 0xAD, 0xE5, 0xB8, 0x83, 0xE6, 0x93, 0xA6, 0xE6, 0x9D, 0x90, 0xE9,
    0x87, 0x87, 0xE5, 0xBD, 0xA9, 0xE8, 0x8D, 0x89, 0xE6, 0xB5, 0x8B, 0xE7,
    0xAD, 0x96, 0xE5, 0xB7, 0xAE, 0xE4, 0xBA, 0xA7, 0xE9, 0x98, 0x90, 0xE5,
    0x9C, 0xBA, 0xE6, 0x89, 0xBF, 0xE5, 0x85, 0x85, 0xE9, 0x85, 0xAC, 0xE4,
    0xBC, 0xA0, 0xE5, 0x88, 0x9B, 0xE8, 0xAF, 0x8D, 0xE6, 0xAC, 0xA1, 0xE4,
    0xBB, 0x8E, 0xE6, 0x9D, 0x91, 0xE5, 0xAD, 0x98, 0xE8, 0xBE, 0xBE, 0xE6,
    0x89, 0x93, 0xE5, 0xA4, 0xA7, 0xE5, 0xBE, 0x85, 0xE6, 0x8B, 0x85, 0xE8,
    0x80, 0xBD, 0xE4, 0xBD, 0x86, 0xE5, 0xBD, 0x93, 0xE6, 0xA1, 0xA3, 0x0D,
    0x0A, 0xE5, 0xAF, 0xBC, 0xE5, 0x88, 0xB0, 0xE9, 0x81, 0x93, 0xE5, 0xBE,
    0xB7, 0xE7, 0xAD, 0x89, 0xE4, 0xBD, 0x8E, 0xE5, 0x9C, 0xB0, 0xE7, 0xAC,
    0xAC, 0xE5, 0x85, 0xB8, 0xE7, 0x82, 0xB9, 0xE7, 0x94, 0xB5, 0xE8, 0xAE,
    0xA2, 0xE5, 0xAE, 0x9A, 0xE5, 0x8A, 0xA8, 0xE9, 0x83, 0xBD, 0xE5, 0xBA,
    0xA6, 0xE7, 0x9F, 0xAD, 0xE6, 0xAE, 0xB5, 0xE5, 0xAF, 0xB9, 0xE5, 0xA4,
    0x9A, 0xE5, 0xA4, 0xBA, 0xE5, 0x84, 0xBF, 0xE8, 0x80, 0x8C, 0xE4, 0xBA,
    0x8C, 0xE6, 0xB3, 0x95, 0xE8, 0x8C, 0x83, 0xE6, 0x96, 0xB9, 0xE6, 0x94,
    0xBE, 0xE9, 0x9D, 0x9E, 0xE5, 0x88, 0x86, 0xE4, 0xBB, 0xBD, 0xE9, 0xA3,
    0x8E, 0xE7, 0xA6, 0x8F, 0xE5, 0xA4, 0x8D, 0xE6, 0x94, 0xB9, 0xE6, 0xA6,
    0x82, 0xE6, 0x84, 0x9F, 0xE5, 0xB9, 0xB2, 0xE5, 0x91, 0x8A, 0xE6, 0xA0,
    0xBC, 0x0D, 0x0A, 0xE4, 0xB8, 0xAA, 0xE5, 0x90, 0x84, 0xE7, 0xBB, 0x99,
    0xE6, 0xA0, 0xB9, 0xE6, 0x9B, 0xB4, 0xE5, 0xB7, 0xA5, 0xE5, 0x85, 0xAC,
    0xE4, 0xBE, 0x9B, 0xE7, 0x8B, 0x97, 0xE6, 0x9E, 0x84, 0xE8, 0xB4, 0xAD,
    0xE5, 0xA4, 0x9F, 0xE5, 0x8F, 0xA4, 0xE5, 0x9B, 0xBA, 0xE6, 0x95, 0x85,
    0xE9, 0xA1, 0xBE, 0xE8, 0xA7, 0x82, 0xE7, 0xAE, 0xA1, 0xE5, 0xB9, 0xBF,
    0xE8, 0xA7, 0x84, 0xE6, 0xA3, 0x8D, 0xE5, 0x9B, 0xBD, 0xE6, 0x9E, 0x9C,
    0xE8, 0xBF, 0x87, 0xE8, 0xBF, 0x98, 0xE5, 0x90, 0xAB, 0xE9, 0x9F, 0xA9,
    0xE5, 0xA5, 0xBD, 0xE5, 0x90, 0x88, 0xE4, 0xBD, 0x95, 0xE5, 0x92, 0x8C,
    0xE6, 0xA0, 0xB8, 0xE5, 0xBE, 0x88, 0xE5, 0x90, 0x8E, 0xE5, 0x80, 0x99,
    0xE5, 0xBF, 0xBD, 0xE6, 0xB9, 0x96, 0xE5, 0x88, 0x92, 0xE6, 0xBB, 0x91,
    0xE5, 0x8C, 0x96, 0x0D, 0x0A, 0xE7, 0x94, 0xBB, 0xE5, 0xBE, 0x8A, 0xE9,
    0xBB, 0x84, 0xE6, 0x83, 0xA0, 0xE6, 0xB4, 0xBB, 0xE8, 0x8E, 0xB7, 0xE7,
    0xA5, 0xB8, 0xE5, 0x9F, 0xBA, 0xE5, 0x8F, 0x8A, 0xE9, 0x9B, 0x86, 0xE8,
    0xBE, 0x91, 0xE5, 0xB7, 0xB1, 0xE8, 0xAE, 0xA1, 0xE9, 0x99, 0x85, 0xE5,
    0xAD, 0xA3, 0xE6, 0x97, 0xA2, 0xE4, 0xBD, 0xB3, 0xE5, 0xAE, 0xB6, 0xE5,
    0x81, 0x87, 0xE4, 0xBB, 0xB7, 0xE9, 0x97, 0xB4, 0xE5, 0x87, 0x8F, 0xE5,
    0x89, 0xAA, 0xE8, 0xA7, 0x81, 0xE4, 0xBB, 0xB6, 0xE5, 0xBB, 0xBA, 0xE8,
    0xB7, 0xB5, 0xE5, 0xB0, 0x86, 0xE9, 0x99, 0x8D, 0xE4, 0xBA, 0xA4, 0xE8,
    0xA7, 0x92, 0xE8, 0xA7, 0x89, 0xE8, 0xBE, 0x83, 0xE9, 0x98, 0xB6, 0xE8,
    0x8A, 0x82, 0xE7, 0xBB, 0x93, 0xE6, 0x88, 0xAA, 0xE7, 0x95, 0x8C, 0xE5,
    0x80, 0x9F, 0xE9, 0x87, 0x91, 0x0D, 0x0A, 0xE4, 0xBB, 0x85, 0xE7, 0xB4,
    0xA7, 0xE5, 0xB0, 0xBD, 0xE8, 0xBF, 0x91, 0xE7, 0xBB, 0x8F, 0xE7, 0xB2,
    0xBE, 0xE6, 0x99, 0xAF, 0xE5, 0xBE, 0x84, 0xE9, 0x95, 0x9C, 0xE5, 0xB0,
    0xB1, 0xE5, 0xB1, 0x80, 0xE5, 0xB7, 0xA8, 0xE5, 0x85, 0xB7, 0xE5, 0x89,
    0xA7, 0xE6, 0x8D, 0xAE, 0xE5, 0x86, 0xB3, 0xE5, 0x9D, 0x87, 0xE7, 0x9C,
    0x8B, 0xE6, 0x8A, 0x97, 0xE8, 0x80, 0x83, 0xE6, 0x89, 0xA9, 0xE6, 0x9D,
    0xA5, 0xE8, 0x80, 0x81, 0xE4, 0xB9, 0x90, 0xE7, 0xB1, 0xBB, 0xE7, 0xA4,
    0xBC, 0xE9, 0x87, 0x8C, 0xE7, 0x90, 0x86, 0xE5, 0x8A, 0x9B, 0xE4, 0xBE,
    0x8B, 0xE8, 0x81, 0x94, 0xE4, 0xB8, 0xA4, 0xE9, 0x87, 0x8F, 0xE5, 0x88,
    0x97, 0xE4, 0xB8, 0xB4, 0xE9, 0xA2, 0x86, 0xE5, 0x8F, 0xA6, 0xE6, 0xB5,
    0x81, 0xE9, 0xBE, 0x99, 0xE8, 0xA3, 0xB8, 0x0D, 0x0A, 0xE8, 0x99, 0x91,
    0xE7, 0x8E, 0x87, 0xE7, 0x95, 0xA5, 0xE4, 0xB9, 0xB0, 0xE5, 0x8D, 0x96,
    0xE5, 0xBF, 0x99, 0xE8, 0xB2, 0x8C, 0xE6, 0xAF, 0x8F, 0xE7, 0xBE, 0x8E,
    0xE9, 0x97, 0xA8, 0xE4, 0xBB, 0xAC, 0xE6, 0xA2, 0xA6, 0xE8, 0xBF, 0xB7,
    0xE5, 0x85, 0x8D, 0xE9, 0x9D, 0xA2, 0xE6, 0x8F, 0x8F, 0xE6, 0xB0, 0x91,
    0xE5, 0x90, 0x8D, 0xE6, 0x98, 0x8E, 0xE6, 0xA8, 0xA1, 0xE6, 0x9F, 0x90,
    0xE6, 0x9C, 0xA8, 0xE7, 0x9B, 0xAE, 0xE5, 0xB9, 0x95, 0xE9, 0x82, 0xA3,
    0xE5, 0xA5, 0x88, 0xE5, 0x8D, 0x97, 0xE9, 0x9A, 0xBE, 0xE5, 0x86, 0x85,
    0xE4, 0xBD, 0xA0, 0xE5, 0xB9, 0xB4, 0xE5, 0xBF, 0xB5, 0xE5, 0xA5, 0xB3,
    0xE5, 0x81, 0xB6, 0xE6, 0x8B, 0x8D, 0xE6, 0x8E, 0x92, 0xE5, 0xBE, 0x98,
    0xE7, 0x89, 0x8C, 0xE5, 0x88, 0xA4, 0xE6, 0x89, 0xB9, 0x0D, 0x0A, 0xE7,
    0xAF, 0x87, 0xE7, 0x89, 0x87, 0xE9, 0xA2, 0x91, 0xE5, 0xB9, 0xB3, 0xE5,
    0xB1, 0x8F, 0xE8, 0xBF, 0xAB, 0xE6, 0x9C, 0xB4, 0xE6, 0x9C, 0x9F, 0xE9,
    0xBD, 0x90, 0xE5, 0x85, 0xB6, 0xE8, 0xB5, 0xB7, 0xE6, 0xB0, 0x94, 0xE7,
    0xAD, 0xBE, 0xE5, 0x89, 0x8D, 0xE9, 0x92, 0xB1, 0xE6, 0xBD, 0x9C, 0xE5,
    0xBC, 0xBA, 0xE5, 0x80, 0xBE, 0xE6, 0xB1, 0x82, 0xE7, 0x90, 0x83, 0xE5,
    0x8F, 0x96, 0xE5, 0x8E, 0xBB, 0xE5, 0x85, 0xA8, 0xE7, 0xA1, 0xAE, 0xE7,
    0x84, 0xB6, 0xE7, 0x83, 0xAD, 0xE4, 0xBA, 0xBA, 0xE8, 0xAE, 0xA4, 0xE4,
    0xBB, 0xBB, 0xE4, 0xBB, 0x8D, 0xE6, 0x97, 0xA5, 0xE5, 0xAE, 0xB9, 0xE5,
    0xA6, 0x82, 0xE5, 0x85, 0xA5, 0xE4, 0xB8, 0x89, 0xE4, 0xBC, 0xA4, 0xE5,
    0x95, 0x86, 0xE4, 0xB8, 0x8A, 0xE5, 0xB0, 0x91, 0xE8, 0xAE, 0xBE, 0x0D,
    0x0A, 0xE6, 0xB6, 0x89, 0xE6, 0x91, 0x84, 0xE8, 0xBA, 0xAB, 0xE6, 0xB2,
    0x88, 0xE5, 0xAE, 0xA1, 0xE7, 0x94, 0x9A, 0xE7, 0x94, 0x9F, 0xE6, 0x96,
    0xBD, 0xE5, 0x8D, 0x81, 0xE5, 0xAE, 0x9E, 0xE7, 0xA4, 0xBA, 0xE5, 0xB8,
    0x82, 0xE5, 0xBC, 0x8F, 0xE4, 0xBC, 0xBC, 0xE4, 0xBA, 0x8B, 0xE8, 0xA7,
    0x86, 0xE6, 0x98, 0xAF, 0xE9, 0x87, 0x8A, 0xE6, 0x94, 0xB6, 0xE6, 0x89,
    0x8B, 0xE9, 0xA6, 0x96, 0xE5, 0x8F, 0x97, 0xE5, 0x94, 0xAE, 0xE6, 0x9A,
    0x91, 0xE7, 0xBD, 0xB2, 0xE6, 0x9C, 0xAF, 0xE6, 0x95, 0xB0, 0xE5, 0x8F,
    0x8C, 0xE6, 0xB0, 0xB4, 0xE9, 0xA1, 0xBA, 0xE5, 0x8F, 0xB8, 0xE6, 0x80,
    0x9D, 0xE8, 0xAE, 0xBC, 0xE8, 0xAF, 0x89, 0xE7, 0xB4, 0xA0, 0xE7, 0xAE,
    0x97, 0xE9, 0x9A, 0x8B, 0xE9, 0x9A, 0x8F, 0xE7, 0xBC, 0xA9, 0xE6, 0x89,
    0x80, 0x0D, 0x0A, 0xE4, 0xBB, 0x96, 0xE5, 0x8F, 0xB0, 0xE8, 0xB0, 0x88,
    0xE5, 0x94, 0x90, 0xE6, 0x8F, 0x90, 0xE9, 0xA2, 0x98, 0xE5, 0xA4, 0xA9,
    0xE6, 0x9D, 0xA1, 0xE8, 0xB0, 0x83, 0xE5, 0xBA, 0xAD, 0xE5, 0x81, 0x9C,
    0xE9, 0x80, 0x9A, 0xE5, 0x90, 0x8C, 0xE5, 0xA4, 0xB4, 0xE6, 0x8A, 0x95,
    0xE5, 0x87, 0xB8, 0xE7, 0xAA, 0x81, 0xE6, 0x8E, 0xA8, 0xE6, 0x89, 0x98,
    0xE6, 0x8B, 0x93, 0xE5, 0xA4, 0x96, 0xE5, 0xAE, 0x8C, 0xE6, 0x99, 0x9A,
    0xE4, 0xBA, 0xA1, 0xE7, 0x8E, 0x8B, 0xE5, 0xBE, 0x80, 0xE5, 0x8D, 0xB1,
    0xE5, 0xBE, 0xAE, 0xE4, 0xB8, 0xBA, 0xE8, 0xBF, 0x9D, 0xE7, 0xBB, 0xB4,
    0xE5, 0xA7, 0x94, 0xE5, 0x8D, 0xAB, 0xE6, 0x9C, 0xAA, 0xE4, 0xBD, 0x8D,
    0xE7, 0xA8, 0xB3, 0xE6, 0x88, 0x91, 0xE4, 0xBA, 0x94, 0xE8, 0xAF, 0xAF,
    0xE6, 0x9E, 0x90, 0x0D, 0x0A, 0xE6, 0x81, 0xAF, 0xE7, 0xB3, 0xBB, 0xE7,
    0xBB, 0x86, 0xE5, 0x85, 0x88, 0xE8, 0xA1, 0x94, 0xE9, 0x99, 0xA9, 0xE5,
    0x8E, 0xBF, 0xE7, 0x8E, 0xB0, 0xE9, 0x99, 0x90, 0xE4, 0xB9, 0xA1, 0xE7,
    0x9B, 0xB8, 0xE5, 0x93, 0x8D, 0xE6, 0x83, 0xB3, 0xE5, 0x90, 0x91, 0xE9,
    0xA1, 0xB9, 0xE8, 0xB1, 0xA1, 0xE5, 0x83, 0x8F, 0xE5, 0xB0, 0x8F, 0xE6,
    0x95, 0x88, 0xE4, 0xBA, 0x9B, 0xE5, 0x86, 0x99, 0xE5, 0xBF, 0x83, 0xE6,
    0x96, 0xB0, 0xE8, 0x96, 0xAA, 0xE4, 0xBF, 0xA1, 0xE6, 0x98, 0x9F, 0xE5,
    0xBD, 0xA2, 0xE5, 0x9E, 0x8B, 0xE6, 0x80, 0xA7, 0xE9, 0x9B, 0x84, 0xE4,
    0xBC, 0x91, 0xE4, 0xBF, 0xAE, 0xE9, 0xA1, 0xBB, 0xE9, 0x9C, 0x80, 0xE8,
    0xAE, 0xB8, 0xE7, 0xBB, 0xAD, 0xE5, 0xAC, 0x9B, 0xE9, 0x80, 0x89, 0xE6,
    0xBC, 0x94, 0xE5, 0xA4, 0xAE, 0x0D, 0x0A, 0xE9, 0x98, 0xB3, 0xE6, 0xA0,
    0xB7, 0xE8, 0xA6, 0x81, 0xE4, 0xB9, 0x9F, 0xE4, 0xB8, 0x9A, 0xE4, 0xB8,
    0x80, 0xE4, 0xBE, 0x9D, 0xE5, 0xAE, 0x9C, 0xE4, 0xBB, 0xA5, 0xE4, 0xB9,
    0x89, 0xE8, 0x89, 0xBA, 0xE4, 0xBA, 0xA6, 0xE5, 0xBC, 0x82, 0xE6, 0x98,
    0x93, 0xE7, 0x9B, 0x8A, 0xE6, 0x84, 0x8F, 0xE5, 0x9B, 0xA0, 0xE9, 0x9F,
    0xB3, 0xE8, 0x8B, 0xB1, 0xE8, 0x90, 0xA5, 0xE5, 0xBD, 0xB1, 0xE5, 0xBA,
    0x94, 0xE6, 0x98, 0xA0, 0xE5, 0xB0, 0xA4, 0xE6, 0x9C, 0x89, 0xE5, 0x8F,
    0x88, 0xE5, 0xA8, 0xB1, 0xE4, 0xB8, 0x8E, 0xE4, 0xBA, 0x88, 0xE7, 0x8B,
    0xB1, 0xE9, 0xA2, 0x84, 0xE9, 0x81, 0x87, 0xE5, 0x85, 0x83, 0xE5, 0x91,
    0x98, 0xE5, 0x8E, 0x9F, 0xE7, 0xBC, 0x98, 0xE7, 0xBA, 0xA6, 0xE6, 0x9C,
    0x88, 0xE9, 0x98, 0x85, 0xE8, 0xB6, 0x8A, 0x0D, 0x0A, 0xE8, 0xBF, 0x90,
    0xE5, 0x93, 0x89, 0xE4, 0xBB, 0x94, 0xE5, 0x86, 0x8D, 0xE9, 0x81, 0xAD,
    0xE5, 0x88, 0x99, 0xE6, 0x8B, 0xA9, 0xE5, 0xA2, 0x9E, 0xE5, 0xB1, 0x95,
    0xE7, 0xAB, 0x99, 0xE5, 0xBC, 0xA0, 0xE9, 0x95, 0xBF, 0xE6, 0x8B, 0x9B,
    0xE6, 0x89, 0xBE, 0xE7, 0x85, 0xA7, 0xE8, 0xBF, 0x99, 0xE7, 0x9D, 0x80,
    0xE9, 0x92, 0x88, 0xE7, 0x9C, 0x9F, 0xE7, 0x94, 0x84, 0xE4, 0xBA, 0x89,
    0xE6, 0x95, 0xB4, 0xE8, 0xAF, 0x81, 0xE6, 0x94, 0xBF, 0xE4, 0xB9, 0x8B,
    0xE5, 0x8F, 0xAA, 0xE7, 0x9F, 0xA5, 0xE7, 0xBB, 0x87, 0xE7, 0x9B, 0xB4,
    0xE8, 0x81, 0x8C, 0xE6, 0x8C, 0x87, 0xE5, 0x88, 0xB6, 0xE8, 0xB4, 0xA8,
    0xE8, 0x87, 0xB4, 0xE4, 0xBC, 0x97, 0xE9, 0x87, 0x8D, 0xE5, 0x91, 0xA8,
    0xE4, 0xB8, 0xBB, 0xE6, 0xB3, 0xA8, 0xE8, 0xA3, 0x85, 0x00
};

// use other ttf file 
TTFConfig ttfcfg( "fonts/game-sc.ttf", 32 );
lbl = Label::createWithTTF( ttfcfg, (char *)rawData );
lbl->enableOutline( Color4B::RED, 2 );
lbl->setPosition( 800, 800 );
scene->addChild( lbl );

auto listener = EventListenerTouchOneByOne::create();
listener->setSwallowTouches( true );
listener->onTouchBegan = [ scene, lbl ]( Touch * t, Event * e )
{
    if( !lbl->getBoundingBox().containsPoint(
        lbl->getParent()->convertToNodeSpace( t->getLocation() )
        ) ) return false;
    xxx( scene, lbl );
    return false;
};
lbl->getEventDispatcher()->addEventListenerWithSceneGraphPriority( listener, lbl );
```

}

void test()
{
    CCLOG( "void test()" );

```
//auto audio = SimpleAudioEngine::getInstance();
auto director = Director::getInstance();
auto sz = director->getWinSize();
auto scene = director->getRunningScene();
if( !scene )
{
    scene = Scene::create();
    director->runWithScene( scene );
}

xxx(scene, nullptr);
```

}

**Comment by shrimpz** [comment]
1, Touch Screen  N Times,
2,  Lock Phone, 
3, Restored Phone,        Crashed!!!!!

if not Crashed , try re-do it.

MyDevices:
HTC 920e
HongMi 1s Telecom version
etc...

**Comment by minggo** [comment]
close this PR since it is outdated and no developers met this issue

**Pull request closed without merging**